src/StyledComponents.js

```diff
@@ -0,0 +1,75 @@
+import { styled } from "@mui/material/styles";
+import Paper from "@mui/material/Paper";
+import customTheme from "./theme";
+import { Box, Button, Divider, Link } from "@mui/material";
+
+const ContactItem = styled(Paper)(({ theme }) => ({
+  padding: theme.spacing(1),
+  textAlign: "center",
+}));
+export default ContactItem;
+
+const HeroTitle = styled(Box)(({ theme }) => ({
+  padding: theme.spacing(1),
+  variant: "h3",
+  background: customTheme.palette.secondary.main,
+  fontSize: 60,
+  border: "none",
+  fontWeight: 600,
+  position: "relative",
+  textAlign: "start",
+  mt: "12%",
+  color: customTheme.palette.primary.main,
+  mb: "5%",
+}));
+export { HeroTitle };
+
+const MenuItemLink = styled(Link)(({ theme }) => ({
+  color: "#fff",
+  fontSize: 20,
+  textDecoration: "none",
+  borderBottom: "2px solid transparent",
+  "&:hover": {
+    borderBottomColor: customTheme.palette.custom.light,
+  },
+  "@media (max-width:675px)": {
```

Comment:
we already have breakpoints in the mui theme, can you please replace it with appropriate one from mui them
like in the example below:
![image](https://github.com/saharSaleh22/Final_ReactJS_Project_GSG/assets/132939680/e18ca638-0e4a-406d-a7b7-eebf9d6d49ba)